corehq/apps/domain/forms.py

```diff
@@ -632,68 +516,146 @@ def save(self, request, domain):
             return False
 
 
-class DomainDeploymentForm(forms.Form):
-    city = CharField(label=ugettext_noop("City"), required=False)
-    countries = forms.MultipleChoiceField(label=ugettext_noop("Countries"),
-            choices=COUNTRIES)
-    region = CharField(label=ugettext_noop("Region"), required=False,
-        help_text=ugettext_noop("e.g. US, LAC, SA, Sub-Saharan Africa, Southeast Asia, etc."))
-    deployment_date = CharField(label=ugettext_noop("Deployment date"), required=False)
-    description = CharField(label=ugettext_noop("Description"), required=False, widget=forms.Textarea)
-    public = ChoiceField(label=ugettext_noop("Make Public?"), choices=tf_choices('Yes', 'No'), required=False)
+def tuple_of_copies(a_list, blank=True):
+    ret = [(item, item) for item in a_list]
+    if blank:
+        ret.insert(0, ('', '---'))
+    return tuple(ret)
+
+
+class PrivacySecurityForm(forms.Form):
+    restrict_superusers = BooleanField(
+        label=_("Restrict Dimagi Staff Access"),
+        required=False,
+        help_text=_("If access to a project space is restricted only users added " +
+                    "to the domain and staff members will have access.")
+    )
+    secure_submissions = BooleanField(
+        label=_("Secure submissions"),
+        required=False,
+        help_text=_(mark_safe(
+            "Secure Submissions prevents others from impersonating your mobile workers."
+            "This setting requires all deployed applications to be using secure "
+            "submissions as well. "
+            "<a href='https://help.commcarehq.org/display/commcarepublic/Project+Space+Settings'>"
+            "Read more about secure submissions here</a>"))
+    )
 
     def save(self, domain):
+        domain.restrict_superusers = self.cleaned_data.get('restrict_superusers', False)
         try:
-            domain.update_deployment(city=self.cleaned_data['city'],
-                countries=self.cleaned_data['countries'],
-                region=self.cleaned_data['region'],
-                date=dateutil.parser.parse(self.cleaned_data['deployment_date']),
-                description=self.cleaned_data['description'],
-                public=(self.cleaned_data['public'] == 'true'))
+            secure_submissions = self.cleaned_data.get(
+                'secure_submissions', False)
+            apps_to_save = []
+            if secure_submissions != domain.secure_submissions:
+                for app in get_apps_in_domain(domain.name):
+                    if app.secure_submissions != secure_submissions:
+                        app.secure_submissions = secure_submissions
+                        apps_to_save.append(app)
+            domain.secure_submissions = secure_submissions
+            domain.save()
+            if apps_to_save:
+                ApplicationBase.bulk_save(apps_to_save)
```

Comment:
This needs to be done like this: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/forms.py#L485-485